components/builder-web/app/actions/users.ts

```diff
@@ -180,9 +181,15 @@ export function setDeletingAccessToken(payload) {
 
 function notifySegment(data: any) {
   const segment = window['analytics'];
-
   if (segment && typeof segment.identify === 'function') {
-    segment.identify(data.id, { email: data.email, name: data.name });
+    segment.identify(data.id, {
```

Comment:
Based upon my prior comment, I suspect this might become `analytics.identify` when it's all said and done. See these Segment docs --> https://segment.com/docs/destinations/heap/#identify